packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -73,12 +96,34 @@ export const SelectablePill = (props: SelectablePillProps) => {
     }
   }
 
-  return (
-    <SelectablePillRoot {...props}>
+  const pillContent = (
+    <>
       {Icon ? <Icon css={iconCss} /> : null}
       <Text variant='body' tag='span'>
         {label}
       </Text>
-    </SelectablePillRoot>
+    </>
   )
+
+  if (type === 'checkbox' || type === 'radio') {
+    return (
+      <label css={[css, { position: 'relative' }]}>
+        <InputRoot
+          {...other}
+          checked={type === 'checkbox' ? isSelected : undefined}
+        />
+        {pillContent}
+      </label>
+    )
+  }
+
+  if (!type || type === 'button' || type === 'reset' || type === 'submit') {
+    return (
+      <button css={css} {...other}>
+        {pillContent}
+      </button>
+    )
+  }
+
+  return null
```

Comment:
this is only when type is set to an unknown type?

if props pass typecheck this is unreachable, right?

Reply:
oh yeah this isn't needed i was just testing something